libraries/RC_Channel/RC_Channel.h

```diff
@@ -144,13 +144,13 @@ class RC_Channel {
         RELAY2 =              34, // Relay2 pin on/off
         RELAY3 =              35, // Relay3 pin on/off
         RELAY4 =              36, // Relay4 pin on/off
-        THROW =               37,  // change to THROW flight mode
-        AVOID_ADSB =          38,  // enable AP_Avoidance library
-        PRECISION_LOITER =    39,  // enable precision loiter
-        AVOID_PROXIMITY =     40,  // enable object avoidance using proximity sensors (ie. horizontal lidar)
-        ARMDISARM =           41,  // arm or disarm vehicle
+        THROW =               37, // change to THROW flight mode
+        AVOID_ADSB =          38, // enable AP_Avoidance library
+        PRECISION_LOITER =    39, // enable precision loiter
+        AVOID_PROXIMITY =     40, // enable object avoidance using proximity sensors (ie. horizontal lidar)
```

Comment:
I'm a bit concerned about re-using IDs.  Someone may have these already configured (and possibly active!) from older options.  Did you check the history of this ID?

Reply:
Rover definitely never had any CH7_OPTION values as high as 40.  It was about 10 or so when we moved to RC_OPTION.